components/popover/index.js

```diff
@@ -42,6 +42,7 @@ class Popover extends Component {
 
```

Comment:
I think the changes here may have broke `master`. All the popovers (inserter, menus...) are centered and not relative to their parent anymore

Reply:
Yep, I see what you mean. I'm currently trying to figure out how my refactor changed that. Hopefully I'll have a fix for you by the time it rolls around to Europe's Monday.

Reply:
Found the bug:
```
		return {
			...rect,
			top: rect.top + topPad,
			bottom: rect.bottom - bottomPad,
			height: rect.height - topPad - bottomPad,
		}
```
In the above `...rect` does not copy across the DomRect's properties as I had expected. The fix will thankfully be simple.

Reply:
The bug should now be fixed.